pkgs/applications/misc/gcstar/default.nix

```diff
@@ -0,0 +1,67 @@
+{stdenv, fetchurl, bash, perl, perlPackages, makeWrapper}: 
+
+stdenv.mkDerivation rec {
+  pname = "gcstar";
+  version = "1.7.1";
+
+  src = fetchurl {
+    url = "https://launchpad.net/${pname}/1.7/${version}/+download/${pname}-${version}.tar.gz";
+    sha256 = "0gcz88slgm14rlsw84gpka7hpdmrdvpdvxp9qvs45gv1383r0b6s";
+  };
+
+  enableParallelBuilding = false;
+  doCheck = false;
+
+ perlDeps = [
+    perl
+    perlPackages.ArchiveZip
+    perlPackages.DateCalc
+    perlPackages.DateTimeFormatStrptime
+    perlPackages.Glib
+    perlPackages.Gtk2
+    perlPackages.GD
+    perlPackages.GDGraph
+    perlPackages.GDText
+    perlPackages.HTMLParser
+    perlPackages.librelative
+    perlPackages.LWPUserAgent
+    perlPackages.Pango
+    perlPackages.XMLSimple
+    perlPackages.XMLParser ];
+
+  buildInputs = [makeWrapper] ++ perlDeps;
+  propagatedBuildInputs = perlDeps;
+
+  installPhase = ''
+    tar xvfz $src
+    cd gcstar
+
+    sed -i 's|/usr/bin/perl|'"${perl}"'/bin/perl|g' bin/gcstar
+    sed -i 's|/usr/bin/perl|'"${perl}"'/bin/perl|g' ./install
+    sed -i 's|/bin/sh|'"${bash}"'/bin/bash|g' share/gcstar/helpers/xdg-open
+
+    perl install --text --prefix=$out
+  '';
+
+  postFixup = "wrapProgram $out/bin/gcstar --prefix PERL5LIB : $PERL5LIB";
+
+  outputs = [ "out" ];
+
+  meta = with stdenv.lib; {
+    homepage = https://launchpad.net/gcstar;
+    description = "GCstar is an application for managing your collections.";
```

Comment:
As per https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md, `meta.description` should:
    * Be capitalized.
    * Not start with the package name.
    * Not have a period at the end.

Reply:
@symphorien thanks for review... I fixed your points and added handling for a desktop file and its icon